resources/articles/selfservice/client-teaser-documentation.md

```diff
@@ -116,3 +116,96 @@ Now, all you need to do is fill in the fields as described above in “Each inpu
 As you may see on the figure above the summary of client teasers are quite simple. You may edit an existing client teaser that will create and upload a new file on our CDN so any previous cached version will hence be refreshed with a new page.
 
 If you have a lot of teasers you also have the ability to simply delete a client teaser, note that if you delete a client teaser it is a **hard deletion** and there is no way for us to get it back. Once deleted, it stays deleted and you will have to create a complete new one if you wish to get it back as it was.
+
+
+# New flow style
+
+![Client teaser example](/images/client_teaser/client_teaser_example.png)
+
+If you are using the new login flow the teaser should not be taller than the login form to the right. 
+You can prevent the teaser from becoming scrollable by adding:
+
+```
+ body {
+     max-width: 100%;
+     max-height: 100%;
+     overflow: hidden;
+ }
+```
+
+
+# Example
+
+In this example we show how to use a background image with a title and some text on top.
+
+Feel free to use this markdown and CSS as a starting point for your new teaser.
+
+#### Teaser markdown:
+
+```
+{.img-bg}
+
+# This is the header {.title}
+
+This is a paragraph. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur feugiat nibh at aliquam tristique. Sed euismod lacus at risus fermentum varius. Sed suscipit, sem vel finibus aliquet, libero nibh efficitur orci, a placerat elit neque a elit.  {.description}
+
+```
+
+#### Teaser CSS:
+
+Notes:
```

Comment:
How about changing to: "Notes to explain comments in the CSS below" or something like that? (I found it confusing when I saw it the first time, since the Notes to the content came before the content itself 😛)